docs/user/reference/juju/juju-roadmap-and-releases.md

```diff
@@ -440,9 +534,12 @@ See the full list in the [milestone page](https://launchpad.net/juju/+milestone/
 
 
 ## :juju: **Juju 3.1**
-> 28 Feb 2025: expected end of security fix support
->
-> 30 July 2023: end of bug fix support
+
+```{caution}
+
+Juju 3.1 series is EOL
```

Comment:
Should we end sentences with a "." like we require for code comments etc.

Reply:
I will leave that for the next update, since we need to refine this whole document anyway.